Tests/UIKitTests/UIButtonExtensionsTests.swift

```diff
@@ -158,6 +158,47 @@ final class UIButtonExtensionsTests: XCTestCase {
         XCTAssertEqual(button.titleEdgeInsets, UIEdgeInsets(top: 0, left: 10, bottom: 0, right: -10))
     }
 
+    func testAlignTextAndImageVertically() {
+        let button = UIButton(frame: CGRect(x: 10, y: 10, width: 100, height: 100))
+        let image = UIImage(color: .green, size: CGSize(width: 10, height: 10))
+        button.setTitleForAllStates("Title")
+        button.setImageForAllStates(image)
+
+        // Image above text
+        button.alignTextAndImageVertically(imageAboveText: true, spacing: 20)
```

Comment:
Create a constant value so that you can reuse it to avoid constantly hard-coding the value (e.g. in the assertions).
```suggestion
        let spacing = CGFloat(20)

        button.alignTextAndImageVertically(imageAboveText: true, spacing: spacing)
```

Reply:
Nice